index.bs

```diff
@@ -36,16 +41,20 @@ urlPrefix: https://tc39.es/ecma262/; spec: ECMASCRIPT
   text: the typed array constructors table; url: #table-49
   text: typed array; url: #sec-typedarray-objects
   text: Number type; url: #sec-ecmascript-language-types-number-type
+  text: %ObjectPrototype%; url: #sec-properties-of-the-object-prototype-object
```

Comment:
These days they would like us to move to `%Object.prototype%`. (Same URL, just change the linking text here and below.)

Reply:
Done.